src/DataService/CachingContextFactory.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace Lullabot\Mpx\DataService;
+
+use phpDocumentor\Reflection\Types\Context;
+
+/**
+ * Creates a context for docblocks, storing cached results for already parsed files.
+ *
+ * @see \phpDocumentor\Reflection\Types\ContextFactory
+ */
+final class CachingContextFactory
+{
+    /** The literal used at the end of a use statement. */
+    const T_LITERAL_END_OF_USE = ';';
+
+    /** The literal used between sets of use statements */
+    const T_LITERAL_USE_SEPARATOR = ',';
+
+    /**
+     * Build a Context given a Class Reflection.
+     *
+     * @param \Reflector $reflector
+     *
+     * @see Context for more information on Contexts.
```

Comment:
This sentence seems redundant/confusing, it could be clearer like this:
`@see phpDocumentor\Reflection\Types\Context for more information on contexts.`

Reply:
:+1: